CHANGELOG.md

```diff
@@ -28,6 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (`JxlDecoderDCOutBufferSize` and `JxlDecoderSetDCOutBuffer`) and will be
   removed in the next release.
 
+## [0.5] - 2021-07-??
+### Changed
+ - Add JxlDecoderRewind and JxlDecoderSkipFrames to decoder API to skip more
```

Comment:
This looks like it should go in the "Add" subsection (this is new stuff).

Reply:
Done